partnercentersellv1/partner_center_sell_v1.go

```diff
@@ -42,7 +42,7 @@ type PartnerCenterSellV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://product-lifecycle.api.test.cloud.ibm.com/openapi/v1"
+const DefaultServiceURL = "http://localhost:5005/openapi/v1"
```

Comment:
The default service URL should come directly from the "servers" field in the API definition.  It should reflect the production endpoint for the service.